draft-ietf-rats-architecture.md

```diff
@@ -877,7 +878,41 @@ are signed.  For example, values might have been generated at boot, and then
 used in claims as long as the signer can guarantee that they cannot have changed
 since boot.
 
-A more detailed discussion with examples appears in {{time-considerations}}.
+The term freshness is applied to both Evidence and Attestation Results in
+this document. Unfortunately, the term freshness is also a bit overloaded and
+in order to illustrate the application of the term in this document better,
+two characteristics that can be part of freshness are elaborated on below:
+recentness and uniqueness:
+
+Recentness:
+
+: a quality of Evidence that is based on the time that has passed between
+creation of the evidence by an Attester (typically by "sealing" a Claims set
+via signing) and the reception of that Evidence by a Verifier or (relaying
```

Comment:
Did you really mean "relaying" party?  This looks like a typo for Relying Party, but Relying Parties get Attestation Results not Evidence per se.   If you really do mean "relaying" party then better to use a term other than "party" ("entity"?) with it so it wouldn't look like a typo.